keyboards/genone/eclipse_65/keymaps/default/keymap.c

```diff
@@ -0,0 +1,21 @@
+#include QMK_KEYBOARD_H
```

Comment:
Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

Reply:
Thanks for the comment.
Added a GPL2+ license header in the keymap.c file.